testsupport/space.go

```diff
@@ -193,7 +193,11 @@ func verifyResourcesProvisionedForSpace(t *testing.T, hostAwait *wait.HostAwaiti
 	}
 
 	// get NSTemplateSet
-	nsTmplSet, err := targetCluster.WaitForNSTmplSet(t, spaceName, wait.UntilNSTemplateSetHasTier(tier.Name), wait.UntilNSTemplateSetHasConditions(Provisioned()))
+	nsTmplSet, err := targetCluster.WaitForNSTmplSet(t, spaceName,
+		wait.UntilNSTemplateSetHasTier(tier.Name),
+		wait.UntilNSTemplateSetHasConditions(Provisioned()),
+		wait.UntilNSTemplateSetProvisionedNamespacesIsNotEmpty(),
```

Comment:
If we go with the suggestion from https://github.com/codeready-toolchain/toolchain-e2e/pull/667/files#r1105280439 then would we still need this check?